features/default_app.feature

```diff
@@ -26,4 +26,4 @@ Scenario: Single-module default app builds successfully
     And the part "versionCode" for request 1 equals "1"
     And the part "versionName" for request 1 equals "1.0"
     And the part "appId" for request 1 equals "com.bugsnag.android.example"
-
+    And the part "overwrite" for request 1 is null
```

Comment:
This line doesn't seem to have any defined steps.

Also the Gemfile is looking for maze-runner locally at the moment.